addons/ctrl-click-run-scripts/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Ctrl+Click to run scripts",
-  "description": "Only run scripts on click if also holding the Ctrl key.",
+  "description": "Only execute clicked blocks and scripts if also holding the Ctrl key or, optionally, if double clicked.",
```

Ctrl+Click still works with the setting enabled.
```suggestion
  "description": "Only execute clicked blocks and scripts if also holding the Ctrl key and, optionally, if double clicked.",
```